File: PYTHON_10AM/JUL02/one.py
class Account:
    def _init_(self,id,name,amount):
        self.acc_id=id 
        self.acc_name=name 
        self.acc_bal=amount

    def deposit_amount(self,amount):
        self.acc_bal=self.acc_bal+amount

a1=Account(101,'Rahul',5000)
a1.deposit_amount(100)
a1.deposit_amount(200)

a2=Account(102,'Sonia',15000)
a2.deposit_amount(2500)
print(a1._dict_)
print(a2._dict_)    

File: java script/JS FUNCTIONS/CLASS FUNCTION/one.js
class Account{
    acc_id;
    acc_Name;
    acc_Bal;
    constructor(id, name, amount){
        this.acc_id = id;
        this.acc_Name = name;
        this.acc_Bal = amount;
    }
}
let rg=new Account(101,"Rahul", 5000);
let pg=new Account(102, "Priyanka", 6000);
let sg=new Account(103,"Sonia",7000);

console.log(rg);
console.log(pg);
console.log(sg);